internal/profile/_static/docker-compose-stack.yml

```diff
@@ -124,7 +125,8 @@ services:
         condition: service_healthy
     healthcheck:
       test: "elastic-agent status"
-      retries: 180
+      start_period: 60s
+      retries: 360
```

Comment:
Even using these settings, there are some cases where elastic_agent_is_ready container still has unhealthy status :disappointed: 
`ERROR: for elastic-agent_is_ready  Container "066fd24a8fc6" is unhealthy.`

Not sure if at least those will help us to make it a bit better 

I've not been able to reproduce it locally to check what it is the output of the healthchecks using `docker inspect`